packages/appsync-emulator-serverless/bin/server.js

```diff
@@ -54,6 +61,15 @@ const main = async () => {
   const serverless = path.join(path.dirname(pkgPath), 'serverless.yml');
   const server = await createServer({ serverless, port, dynamodb });
   console.log('started at url:', server.url);
+  dynamodb_port && console.log(
```

Comment:
I'm not sure if its worthwhile to print all of this.  Perhaps we can use what's in #23 ?